lib/mix/lib/mix/local/installer.ex

```diff
@@ -111,8 +112,11 @@ defmodule Mix.Local.Installer do
           """
       end
 
+      case module.after_install(dst_file_path, elem(file, 1), previous_files) do
```

Comment:
I think this should go into `{:ok, binary} ->` clause, this `elem(file, 1)` looks indirect and we raise in all other clauses.